src/main/java/org/broadinstitute/hellbender/tools/copynumber/plotting/PlotModeledSegments.java

```diff
@@ -0,0 +1,233 @@
+package org.broadinstitute.hellbender.tools.copynumber.plotting;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.copynumber.allelic.alleliccount.AllelicCount;
+import org.broadinstitute.hellbender.tools.copynumber.allelic.alleliccount.AllelicCountCollection;
+import org.broadinstitute.hellbender.tools.copynumber.coverage.copyratio.CopyRatio;
+import org.broadinstitute.hellbender.tools.copynumber.coverage.copyratio.CopyRatioCollection;
+import org.broadinstitute.hellbender.tools.copynumber.formats.CopyNumberStandardArgument;
+import org.broadinstitute.hellbender.tools.copynumber.multidimensional.model.ModeledSegment;
+import org.broadinstitute.hellbender.tools.copynumber.multidimensional.model.ModeledSegmentCollection;
+import org.broadinstitute.hellbender.utils.R.RScriptExecutor;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.io.Resource;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Plot segmented copy-ratio and minor-allele-fraction results.
+ *
+ * <p>The order and representation of contigs in plots follows the contig ordering within the required reference sequence dictionary. </p>
+ *
+ * <h3>Example</h3>
+ *
+ * <pre>
+ * gatk-launch --javaOptions "-Xmx4g" PlotACNVResults \
```

Comment:
IS this command line correct?  `PlotACNVResults`?

Reply:
Good catch.

Reply:
I also need to update the rest of this javadoc.

Reply:
Updated the javadoc for PlotDenoisedCopyRatios as well.